.mdlrc.style.rb

```diff
@@ -0,0 +1,7 @@
+# https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
+
+# Start with all built-in rules.
+all
+
+rule 'MD013', :line_length => -1
+rule 'MD030', :ol_multi => 2
```

Comment:
These can only be configured in a Ruby source file? Or would it still be possible to have it use a configuration file?

Reply:
Yes rules must be a ruby source file: https://github.com/markdownlint/markdownlint/blob/master/docs/creating_rules.md

There is also the `.mdlrc` but that's just to define command line flags as config: https://github.com/markdownlint/markdownlint/blob/master/docs/configuration.md#mdl-configuration

Reply:
Something like this:

```dockerfile
# mdl is a lint tool for markdown files
FROM gem AS mdl
COPY <<-"EOT" /tmp/mdlrc.style.rb
	# https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md
	# Start with all built-in rules.
	all
	rule 'MD013', :line_length => -1
	rule 'MD030', :ol_multi => 2
EOT
RUN --mount=type=bind,target=. \
  mdl --rules /tmp/mdlrc.style.rb .
```

Reply:
hm... that's a pity. I was hoping we could keep "config" and "code" a bit separate (and a JSON or YAML, or TOML file may be less "daunting' than editing a source-code file - and possibly easier to discover)

Reply:
yeah that's something we have to deal with also in js like https://github.com/docker/build-push-action/blob/master/jest.config.ts